conda/recipes/ucx-py/meta.yaml

```diff
@@ -39,7 +39,7 @@ requirements:
 
 test:
   requires:
-    - cudatoolkit {{ cuda_version }}.*
+    - cuda-version {{ cuda_version }}.*
```

Comment:
```suggestion
    - cuda-version ={{ cuda_version }}
```

Reply:
Done in https://github.com/rapidsai/ucx-py/pull/967/commits/c04f3de0c44f6140f8cd522ea7368f6283ff908c .